apollo-router-core/src/spec/schema.rs

```diff
@@ -750,6 +757,15 @@ mod tests {
             .is_none());
     }
 
+    #[test]
+    fn schema_id() {
+        let schema = Schema::from_str(include_str!("../testdata/contract_schema.graphql")).unwrap();
+        assert_eq!(
+            schema.schema_id,
+            Some("6881633761291F4A6E4F9A4A6BBE7AD69A80C63EBEBFC357464B7A6EF276EF0A".to_string())
```

Comment:
Bah, I meant to say this and forgot — lower case hex :)

Reply:
[c70be5b](https://github.com/apollographql/router/pull/1020/commits/c70be5bf3f3fc752f2ab923a42bbe25d1f7c3d94)